README.md

```diff
@@ -206,6 +206,14 @@ to configure multiple supported headers, you can set it like below:
 cf set-env <YOUR_APP> HTTP_RESPONSE_HEADERS '{"Referrer-Policy": "no-referrer-when-downgrade", "X-Content-Type-Options": "nosniff"}'
 ```
 
+### Enabling SameSite / Secure Cookie Header Injection for Runtime < 8.11
+
+Google Chrome will - at a certain moment - [enforce cookie security](https://www.chromium.org/updates/same-site) by requiring the `SameSite` and `Secure` atrributes for all cookies. Mendix runtime versions < 8.11 do not include these properties in cookies.
+
+The buildpack can inject these two properties into all cookies for affected runtime versions.
+
+This workaround is disabled by default. If your application supports injecting these cookies, you can choose to enable cookie header injection by setting the `SAMESITE_COOKIE_WORKAROUND` environment variable to `true`.
```

Comment:
Why would we actually have the runtime version check, if this fix is opt-in?

Reply:
The workaround is needed for 8.11 as well, because of a client change we're doing for 8.12, but that could be part of the documentation.

Reply:
We don't want people enabling this and when 8.12 arrives struggling with finding why their custom cookie settings in 8.12 are not working.

Reply:
I'll change the 8.11 to 8.12.